paradise.dme

```diff
@@ -782,6 +782,7 @@
 #include "code\game\machinery\camera\camera_presets.dm"
 #include "code\game\machinery\camera\motion.dm"
 #include "code\game\machinery\camera\tracking.dm"
+#include "code\game\machinery\computer\arcade_games\recruiter"
```

Comment:
You manually put it here, instead of just ticking the file, making you forget to add `.dm` to the end of it

Reply:
Me when github lies to me
You can resolve this